py/torch_tensorrt/dynamo/conversion/_TRTInterpreter.py

```diff
@@ -619,27 +609,32 @@ def run(
             builder_config, self.compilation_settings.timing_cache_path
         )
 
-        serialized_engine = self.builder.build_serialized_network(
+        # if strip_engine_weights is true, the serialized engine need to be refitted before using
+        maybe_unrefitted_serialized_engine = self.builder.build_serialized_network(
```

Comment:
Why is this maybe unrefitted engine?

Reply:
please see the design in the comment below. If compilation_settings.strip_engine_weights is true, it needs to be refitted, else it doesn't. so it's _maybe_